notebooks/loading-and-processing-data.ipynb

```diff
@@ -615,7 +569,7 @@
DoPBoNzvj3/8Iz766CO8++67WLt2LVpbW3HeeefBarUq9+Fza2AJx/MK4GsWufvuu+9wyy23YMOGDVixYgUsFgvOOusst+fNE088gaeffhqLFy/Gjz/+iNzcXMyYMUM5axOIgtcsmboEgPzRRx+53TZmzBj54Ycfdrtt4sSJ8r333qt8fPrpp8u33Xabz+v+8ssvMgB5w4YNym3r16+XAci7du0Ky9opunV+bu3evVsGIG/fvl25zWKxyOnp6fLLL78sy7IsNzU1yVqtVn733XeV+1RUVMgqlUr+8ssvZVnmc2ug68rzSpb5mkWB1dTUyADk7777TpZlWbbZbHJubq7897//XblPR0eHnJKSIi9ZskSW5eh4zWIGKIxOOeUUfPLJJ6ioqIAsy1i5ciX27NmDs88+2+1+b731FjIzMzFmzBjcddddbhHx+vXrkZKSghNOOEG57cQTT0RKSgrWrVvXY98LRQ+j0QgAiI2NVW5Tq9WIiYnB2rVrAQA//fQTzGYzzjrrLOU++fn5GDt2rPK84XOLXAXzvBL4mkX+NDc3AwDS09MBAAcPHkR1dbXb65FOp8Ppp5+uPCei4TWLh6GG0T//+U/85je/QWFhITQaDVQqFV555RWccsopyn2uvvpqDB48GLm5udi+fTsWLFiAn3/+GStWrAAAVFdXIzs72+Pa2dnZqK6u7rHvhaLHyJEjUVJSggULFuDFF19EQkICnn76aVRXV6OqqgqA/XkTExODtLQ0t8fm5OQozxs+t8hVMM8rgK9Z5J8sy7jjjjtwyimnYOzYsQCg/Lvn5OS43TcnJwdlZWXKfXr7NYsBUBj985//xIYNG/DJJ5+gpKQEq1evxu9//3vk5eVh+vTpAOy1dGHs2LEYPnw4jj/+eGzevBkTJ04EAEiS5HFtWZa93k79n1arxQcffIAbbrgB6enpUKvVmD59OmbNmhXwsZ2fN3xukRDs84qvWeTP/PnzsXXrVo+sIeD5vAjmOdGTr1ksgYVJe3s77rnnHjz99NOYPXs2xo0bh/nz5+Pyyy/HP/7xD5+PmzhxIrRarbLzIjc3F0ePHvW4X21trUc0TQPHpEmTUFpaiqamJlRVVeHLL79EfX09Bg8eDMD+vDGZTGhsbHR7XE1NjfK84XOLOgv0vPKGr1kk/OEPf8Ann3yClStXorCwULk9NzcXADyyNJ1fj3r7NYsBUJiYzWaYzWaoVO4/UrVaDZvN5vNxO3bsgNlsRl5eHgBgypQpaG5uxg8//KDcZ+PGjWhubsZJJ50UmcVTn5GSkoKsrCzs3bsXmzZtwgUXXADA/kam1WqVsgQAVFVVYfv27crzhs8t8sXX88obvmaRLMuYP38+PvzwQ3z77bceAbMombq+HplMJnz33XfKcyIqXrO63UY9gOj1ennLli3yli1bZADy008/LW/ZskUuKyuTZdm+W2LMmDHyypUr5QMHDsivvfaaHBsbKz///POyLMvyvn375Iceekj+8ccf5YMHD8qfffaZPHLkSHnChAmyxWJRvs7MmTPlcePGyevXr5fXr18vH3vssfJ5553XK98z9YxAz63//ve/8sqVK+X9+/fLy5Ytk0tKSuSLL77Y7Rq/+93v5MLCQvnrr7+WN2/eLJ9xxhny+PHj+dwawLr7vOJrFnlz8803yykpKfKqVavkqqoq5U9bW5tyn7///e9ySkqK/OGHH8rbtm2Tr7zySjkvL09uaWlR7tPbr1kMgEKwcuVKGYDHn2uvvVaWZVmuqqqSr7vuOjk/P1+OjY2VR4wYIT/11FOyzWaTZVmWy8vL5dNOO01OT0+XY2Ji5KFDh8q33nqrXF9f7/Z16uvr5auvvlpOSkqSk5KS5KuvvlpubGzs4e+WelKg59azzz4rFxYWylqtVi4uLpbvvfde2Wg0ul2jvb1dnj9/vpyeni7HxcXJ5513nlxeXu52Hz63BpbuPq/4mkXeeHtOAZBfe+015T42m01+4IEH5NzcXFmn08mnnXaavG3bNrfr9PZrluT4ZoiIiIgGDPYAERER0YDDAIiIiIgGHAZARERENOAwACIiIqIBhwEQERERDTgMgIiIiGjAYQBEREREAw4DICLqdx588EEcd9xxvb0MIopiHIRIRH1KoFOgr732WixevBhGoxEZGRk9tCoi6msYABFRn+J6wvR//vMf3H///di9e7dyW1xcHFJSUnpjaUTUh7AERkR9Sm5urvInJSUFkiR53Na5BHbdddfhwgsvxGOPPYacnBykpqbioYcegsViwd1334309HQUFhbi1VdfdftaFRUVuPzyy5GWloaMjAxccMEFOHToUM9+w0QUEQyAiGhA+Pbbb1FZWYnVq1fj6aefxoMPPojzzjsPaWlp2LhxI373u9/hd7/7HQ4fPgwAaGtrw7Rp05CYmIjVq1dj7dq1SExMxMyZM2EymXr5uyGi7mIAREQDQnp6Ov75z39ixIgRmDdvHkaMGIG2tjbcc889GD58OBYsWICYmBh8//33AIB3330XKpUKr7zyCo499liMGjUKr732GsrLy7Fq1are/WaIqNs0vb0AIqKeMGbMGKhUzt/5cnJyMHbsWOVjtVqNjIwM1NTUAAB++ukn7Nu3D0lJSW7X6ejowP79+3tm0UQUMQyAiGhA0Gq1bh9LkuT1NpvNBgCw2WyYNGkS3nrrLY9rZWVlRW6hRNQjGAAREXkxceJE/Oc//0F2djaSk5N7ezlEFGbsASIi8uLqq69GZmYmLrjgAqxZswYHDx7Ed999h9tuuw1Hjhzp7eURUTcxACIi8iI+Ph6rV69GcXExLr74YowaNQrz5s1De3s7M0JE/QAHIRIREdGAwwwQERERDTgMgIiIiGjAYQBEREREAw4DICIiIhpwGAARERHRgMMAiIiIiAYcBkBEREQ04DAAIiIiogGHARARERENOAyAiIiIaMBhAEREREQDDgMgIiIiGnD+H2uduVoMpa5xAAAAAElFTkSuQmCC\n",
```

Comment:
what in the name of Odin is this massive turd? :rofl:

Reply:
an image

Reply:
oh no! I thought it's secret code to communicate with the aliens :alien: But can we not store 12 million lines of `base64` and just produce the image on the fly? At any rate, not about this PR

Reply:
That would mean downloading the data from ESGF every time we build our documentation, which is a lot more trouble..